openhgnn/models/HMPNN.py

```diff
@@ -0,0 +1,92 @@
+import dgl
+import torch as th
+import torch.nn as nn
+import torch.nn.functional as F
+import dgl.nn as dglnn
+from . import BaseModel, register_model
+
+
+@register_model("HMPNN")
+class HMPNN(BaseModel):
+    r"""
+    Heterogeneous Message-Passing Neural Network (HMPNN)
+    A simple implementation of HMPNN from paper for experimenting in KTN.
+    Parameters
```

Comment:
There should be a blank line between line 13 and 14. Please refer to the grammar of `rst`.